src/components/HomePage/MoviesCounterComponent.tsx

```diff
@@ -1,19 +1,16 @@
-import * as React from 'react';
+import React, { FC } from 'react';
 
-const MoviesCounterComponent: React.FC<{}> = () => {
+export const MoviesCounterComponent: FC = () => {
   const moviesCounter = '15';
   // mock data, later it will be data from (db results)
 
   return (
     <p className="mt-2">
       <strong id="movies-counter" className="fw-bold">
         {moviesCounter}
-        {' '}
       </strong>
       {' '}
       movies found
```

Comment:
`{' movies found'}` - should work

Reply:
Thanks. Fixed in commit 10558d1d1e5a9dd451890f797bd08d7b40d79eba